target/arc/semfunc-v3.c

```diff
@@ -178,7 +178,7 @@ arc_gen_FLAG (DisasCtxt *ctx, TCGv src)
   ReplMask(status32, src, temp_28);
   gen_set_label(done_4);
   gen_set_label(done_2);
-  setRegister(R_STATUS32, status32);
+  gen_helper_set_status32(cpu_env, status32);
```

Comment:
Wherever you see `gen_helper_SET_status32()`, be that as it may in semfunc-v2 or semfunc-v3, the return should be come `DISAS_UPDATE`.